spirv_hlsl.hpp

```diff
@@ -114,6 +114,10 @@ class CompilerHLSL : public CompilerGLSL
 		// By default, a readonly storage buffer will be declared as ByteAddressBuffer (SRV) instead.
 		bool force_storage_buffer_as_uav = false;
 
+		// Similar to force_storage_buffer_as_uav but only forces set bindings.
+		// Use set_hlsl_force_storage_buffer_as_uav to specify the binding by ID.
+		bool force_storage_buffer_as_uav_by_id = false;
```

Comment:
I'm not sure why this needs to be a separate option. If you call set_hlsl_force_storage_buffer_as_uav that should be enough.